### ISSUE TITLE
Create consistent thumbnails for library files

### ISSUE THREAD

**Issue by SeamusLeonardHPE** [grommet/hpe-design-system#3781]
Ladders up to 2024 Q2 Objective 4

**Create thumbnail templates for (in order of priority)**
Component variants should be created in the "utility component" file, that file should be available only by DS team members.

- [ ] Library
- [ ] Assets
- [ ] Archive
- [ ] Sandbox
- [ ] Working files
- [ ] Office Hours (Low)

[WIP file](https://www.figma.com/file/uq6VTBivgi4d1nwDziraYx/%233781---Create-consistent-thumbnails-for-library-files?type=design&node-id=1%3A12&mode=design&t=p13PJq0LhfOmO1yx-1)

**Comment by ashifalinadaf** [comment]
Based on the discussion had with Seamus, Cillian, Oliver and I: 
It is concluded that we have to have a broader discussion on the management of the files for Design System library, before jumping into the thumbnail part. 

Segregating the work between,  
1. "Public facing" (Components, Color, Type & Icons) files as a separate task as @oliverHPE wants Brand team to design them.
2. Project folders - How we manage ongoing work, and archive our research rational.

Keeping in mind OKR  description: Consistent thumbnails, author, last modified by, annotations, etc.

Created a[ Figjam file ](https://www.figma.com/file/t962KFrMOABc6VhMO9dDoR/DS---Library-management?type=whiteboard&node-id=0%3A1&t=7VPEhkMdnxELyLl3-1)to have a further discussion and work on managing the Design System files. 

____________________________________

**Had a discussion with Bhim and others on how does he wants to see the DS assets/files.** 

1. They approaches colleagues/ experiences personal to look into appropriate component.
2. Use component directly from the library - if not available look into the site. 
3. Approaching Devs from GLP to see if it works.
4. Anything related to grommet part asking- grommet experts Matt and Taylor or others. 
5. They really don't go deep into the rationale on any design decisions. 
6. Approach DS site only after exhausting all the options.

**Comment by SeamusLeonardHPE** [comment]
Closed as being tracked as outcome for [#3720](https://github.com/grommet/hpe-design-system/issues/3720)
